force-app/main/default/lwc/platformDevCertCalculator/platformDevCertCalculator.js

```diff
@@ -0,0 +1,91 @@
+import { LightningElement, track } from 'lwc';
+const devFundWeight = 0.23;
+const processAutoWeight = 0.3;
+const userInterfaceWeight = 0.25;
+const testDebugWeight = 0.22;
+
+export default class PlatformDevCertCalculator extends LightningElement {
+
+    devFundamentals = 50;
+    processAutomation = 50;
+    userInterface = 50;
+    testDebugDeploy = 50;
+
+    certificationScore = 90;
+    numberOfQuestionsCorrect = 60;
+
+    showResources = false;
+
+    currentHistoryId = 0;
+
+    @track attemptHistory = [
+        {Id: 1, Score:50},
+        {Id: 2, Score:25}
+    ];
+
+    calculateScore(){
+        let devFundWeightScore = this.devFundamentals * devFundWeight;
+        let processAutoWeightScore = this.processAutomation * processAutoWeight;
+        let userInterfaceWeightScore = this.userInterface * userInterfaceWeight;
+        let testDebugWeightScore = this.testDebugDeploy * testDebugWeight;
+
+        this.certificationScore = devFundWeightScore + processAutoWeightScore + userInterfaceWeightScore + testDebugWeightScore;
+        console.log(this.certificationScore);
+
+        this.showResourcesIfFailed();
+        this.addAttemptHistory(this.certificationScore);
+    }
+
+    handleChange(event){
+        console.log(event);
+        console.log(event.target, event.target.name, event.target.value);
+
+        const inputName = event.target.name;
+        let value = Number(event.target.value);
+
+        if(inputName === 'devFundamentals'){
+            this.devFundamentals = value;
+        }else if (inputName === 'processAutomation'){ 
+            this.processAutomation = value;  
+
+        }else if (inputName === 'userInterface'){
+            this.userInterface = value;
+        }else if (inputName === 'testDebugDeploy')  {
+            this.testDebugDeploy = value;
+        }   
+               
+    }
+
+    showResourcesIfFailed(){
+        if(this.certificationScore < 68){
+            this.showResources = true;
+        } else {
+            this.showResources = false;
+        }
+    }
```

Comment:
Try using a getter method for this https://developer.salesforce.com/docs/platform/lwc/guide/js-props-getters-setters.html